buglist.cgi

```diff
@@ -114,15 +116,15 @@ my $format = $template->get_format("list/list", scalar $cgi->param('format'),
 # Safari (WebKit) does not support it, despite a UA that says otherwise (bug 188712)
 # MSIE 5+ supports it on Mac (but not on Windows) (bug 190370)
 #
-my $serverpush =
-  $format->{'extension'} eq "html"
-    && exists $ENV{'HTTP_USER_AGENT'}
-      && $ENV{'HTTP_USER_AGENT'} =~ /Mozilla.[3-9]/
-        && (($ENV{'HTTP_USER_AGENT'} !~ /[Cc]ompatible/) || ($ENV{'HTTP_USER_AGENT'} =~ /MSIE 5.*Mac_PowerPC/))
-          && $ENV{'HTTP_USER_AGENT'} !~ /(?:WebKit|Trident|KHTML)/
-            && !$agent
-              && !defined($cgi->param('serverpush'))
-                || $cgi->param('serverpush');
+my $serverpush = 0;
+  # $format->{'extension'} eq "html"
+  #   && exists $ENV{'HTTP_USER_AGENT'}
+  #     && $ENV{'HTTP_USER_AGENT'} =~ /Mozilla.[3-9]/
+  #       && (($ENV{'HTTP_USER_AGENT'} !~ /[Cc]ompatible/) || ($ENV{'HTTP_USER_AGENT'} =~ /MSIE 5.*Mac_PowerPC/))
+  #         && $ENV{'HTTP_USER_AGENT'} !~ /(?:WebKit|Trident|KHTML)/
+  #           && !$agent
+  #             && !defined($cgi->param('serverpush'))
+  #               || $cgi->param('serverpush');
```

Comment:
is serverpush a TODO item, or is it being dropped by this change?
if it's being dropped you should remove all related code.